api/billingd/service/service.go

```diff
@@ -550,6 +562,18 @@ func getUsage(product Product, total int64) *pb.Usage {
 	}
 }
 
+func getUnits(product Product, total int64) (freeUnits, paidUnits int64) {
```

Comment:
Nit: the named returns were mostly as documentation? (since both are the same type, might make sense). 
Asking since names not used in impl and might be a bit weird.

Reply:
Yep, for documentation, but will clean up the return to use those params so it's less weird.